docs/guides/getting_started.md

```diff
@@ -408,10 +408,30 @@ const links = $('a[href]')
     .map((i, el) => $(el).attr('href'))
     .get();
 
-const ourDomain = 'apify.com';
-const absoluteUrls = links.map(link => new URL(link, ourDomain));
+const {protocol, hostname} = new URL(request.loadedUrl);
+const absoluteUrls = links.map(link => new URL(link, request.loadedUrl));
+
+const sameDomainLinks = absoluteUrls.filter(url => url.protocol === protocol && url.hostname.endsWith(hostname));
```

Comment:
Maybe we can live without the `protocol` check? I don't assume there will be `ftp://` or similar links in most websites and if we do this strict equality, it will not get `http` links from `https` sources and vice versa. We could do a proper `https?` regex but I think that would just be too complicated for this usecase.